src/scripts/pokeballs/Pokeballs.ts

```diff
@@ -74,12 +74,20 @@ class Pokeballs implements Feature {
             }, 1250, 'Increased catch rate on water routes', new RouteKillRequirement(10, GameConstants.Region.hoenn, 101)),
 
             new Pokeball(GameConstants.Pokeball.Lureball, () => {
-                const numLandPokemon = Routes.getRoute(player.region,player.route()).pokemon.land.length > 0;
-                const isWaterPokemon = Routes.getRoute(player.region,player.route()).pokemon.water.includes(Battle.enemyPokemon().name);
+                if (App.game.gameState == GameConstants.GameState.fighting && player.route()) {
+                    const numLandPokemon = Routes.getRoute(player.region,player.route()).pokemon.land.length > 0;
```

Comment:
`numLandPokemon` is not totally correct. Something like `LandPokemonExists` would be more correct. (I know it was there before, you might as well fix the code while you are working with it)

Reply:
changed to `hasLandPokemon`